packages/components/src/components/Radio/Radio.test.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from "react";
+import { render, screen } from "@testing-library/react";
+import { Radio } from "./Radio";
+import { RadioGroup } from "./RadioGroup";
+
+describe("RadioGroup", () => {
+  it("should render a radio group", () => {
+    render(
+      <RadioGroup label="View density">
+        <Radio id="default" label="Default" value="default" />
+      </RadioGroup>
+    );
+    const renderedRadioGroup = screen.getByRole("radiogroup");
+    const renderedRadio = screen.getByRole("radio");
+    expect(renderedRadioGroup).toBeInTheDocument();
+    expect(renderedRadio).toBeInTheDocument();
+  });
+
+  it("should render a radio group with an accessible label", () => {
+    render(
+      <RadioGroup label="View density">
+        <Radio id="default" label="Default" value="default" />
+      </RadioGroup>
+    );
+    const renderedRadioGroup = screen.getByRole("radiogroup");
+    expect(renderedRadioGroup).toHaveAttribute("aria-label", "View density");
+  });
+
+  it("should render a unchecked radio", () => {
+    render(
+      <RadioGroup>
+        <Radio id="default" label="Default" value="default" />
+      </RadioGroup>
+    );
+    const renderedRadio = screen.getByRole("radio");
+    expect(renderedRadio).not.toBeChecked();
+    expect(renderedRadio).toHaveAttribute("data-state", "unchecked");
+  });
+
+  it("should render a checked radio", () => {
+    render(
+      <RadioGroup defaultValue="default">
```

Comment:
```suggestion
      <RadioGroup>
```

Reply:
I changed in the test, and it wasn't needed 🤔

Reply:
`defaultValue` will help with unnecessary React re-renders. It's good to use even in tests. https://reactjs.org/docs/uncontrolled-components.html#default-values